src/test/java/com/checkmarx/flow/cucumber/integration/github/GitHubResolveIssueVulnerabilitiesSteps.java

```diff
@@ -0,0 +1,105 @@
+package com.checkmarx.flow.cucumber.integration.github;
+
+import com.checkmarx.flow.dto.BugTracker;
+import com.checkmarx.flow.dto.Issue;
+import com.checkmarx.flow.dto.ScanRequest;
+import com.checkmarx.flow.exception.ExitThrowable;
+import com.checkmarx.flow.exception.MachinaException;
+import com.checkmarx.sdk.config.Constants;
+import com.checkmarx.sdk.dto.Filter;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+public class GitHubResolveIssueVulnerabilitiesSteps extends GitHubCommonSteps {
+
+    private static final String INPUT_BASE_PATH = "cucumber/data/sample-sast-results/github-results-samples/";
+    private static final String INPUT_VUL_BEFORE_FIX = "5-findings-different-vuln-same-file.xml";
+    private static final String INPUT_VUL_AFTER_FIX = "2-high-findings-same-vuln-same-file-with-not-ex-status.xml";
+    private static final String REPO_NAME = "VB_3845";
+    private static final String BRANCH_NAME = "master";
+    private static final String TEAM_NAME = "CxServer";
+    private static final String NAMESPACE = "cxflowtestuser";
+    private static final int EXPECTED_CODE_LINES_BEFORE_FIX = 2;
+    private static final int EXPECTED_CODE_LINES_after_FIX = 1;
+
+    private ScanRequest scanRequest;
+    private Filter filter;
+
+    @Before("@GitHubResolveIssueVulnerabilities")
```

Comment:
please consider creating a common "GitHub service" with the basic functionality. too many thing in this file are like the previous one